### ISSUE TITLE
Leaking von privaten Informationen

### ISSUE THREAD

**Issue by stlorenz** [bundesAPI/handelsregister#8]
Hat das schon jemand auf dem Schirm:

https://www.heise.de/news/Digitalisierung-Handelsregister-de-verraet-private-Daten-7202516.html

Ist das wirklich ein Thema im endgültigen Datensatz oder wird dieser keine Dokumente und ggf damit auch keine der hier angesprochenen personenbezogenen Daten enthalten?

**Comment by mkschulze** [comment]
Steht zumindest auf der ToDo Liste :D
OCR Scan und man hat schön alles fein sortiert verfügbar.

![Bildschirmfoto 2022-08-05 um 18 32 19](https://user-images.githubusercontent.com/3390820/183121369-13ac781e-f28a-40b7-94f7-b3254b98fcec.png)

**Comment by mkschulze** [comment]
Ja, müsste man dem Gericht mal schreiben, ob die vielleicht auf Antrag etwas schwärzen würden. Aber die sehen halt § 10a HGB vorrangig, da wird sogar die Aufhebung der DSGVO geregelt. https://dejure.org/gesetze/HGB/10a.html

Mir ist auch unwohl dabei meine Adresse und Geburtstdatum als Gesellschafter da zu lesen und ich bezweifle auch den Mehrwert für die Rechtssicherheit im Geschäftsverkehr. Zumindest ein Teil der Daten wäre doch zu schützen, damit man nicht ganz nackt dar steht.

Es macht ja Sinn gucken zu können, mit wem ich es zu tun habe, aber da reichen mir auch weniger Daten würde ich meinen.

Lässt sich auf jedenfall ne schöne App mit bauen, vielleicht in TypeDB. :D

**Comment by mkschulze** [comment]
Ja, stimme ich voll zu. Die Möglichkeiten der Weiterverarbeitung durch so ein Tool wären abzuwägen.

Mit Aufhebung meine ich nur für den Zweck der Veröffentlichung im HR halt, siehe Abs. 1 S. 1:
 
> Das Auskunftsrecht nach Artikel Absatz 1 und das Recht auf Erhalt einer Kopie nach Artikel 15 Absatz 3 der [Verordnung (EU) 679/2016] des Europäischen Parlaments und des Rates vom 27. April 2016 zum Schutz natürlicher Personen bei der Verarbeitung personenbezogener Daten, zum freien Datenverkehr `und zur Aufhebung der` [Richtlinie 95/46/EG](http://eur-lex.europa.eu/legal-content/DE/TXT/?uri=celex:31995L0046&locale=de) (Datenschutz-Grundverordnung) ([ABl. L 119 vom 4.5.2016, S. 1](http://eur-lex.europa.eu/legal-content/DE/AUTO/?uri=uriserv:OJ.L_.2016.119.01.0001.01.DEU&locale=de&toc=OJ:L:2016:119:TOC); L 314 vom 22.11.2016, S. 72) wird dadurch erfüllt, dass die betroffene Person Einsicht in das Handelsregister und in die zum Handelsregister eingereichten Dokumente nehmen kann.

Natürlich nicht generell.

**Comment by LilithWittmann** [comment]
Also ich glaube wir müssen da realistisch bleiben:
Wir werden als Zivilgesellschaftliche Initiative nicht Millionen von Dokumenten bereinigen können. 
Wir haben da eine politische Entscheidung, dass diese Dokumente öffentlich sein sollen - waren sie defacto schon immer.
Gerade Unternehmer*innen sollte bewusst sein, dass diese Dokumente öffentlich sind. Das die Dokumente öffentlich sind und die DSGVO darauf keine Anwendung findet ist in HGB § 10 geregelt.

All die oben genannten Angriffsvektoren sind auch heute bereits ohne Daten über die Eigentümerstrukturen machbar. Und Handelsregister-Dumps ohne Eigentümerstruktur gibt es massenhaft.

Es ist auch halbwegs nutzlos, wenn wir in einer Kopie bei uns die Dokumente bereinigen, während die Originalen weiterhin abrufbar sind. Das schützt weder vor Stalking noch sonst irgendwas. Weil die Originale mit allen Daten sind einen Klick weit weg.

Worüber wir meiner Meinung nach sprechen könnten: Das Vereinsregister auszusparen. Weil dort absehbar Daten von Mitgliedern erfasst wurden, ohne das diese davon überhaupt wussten.

Und die Intressensabwägung zwischen "Transparenz von Eigentümerstrukturen" zu "Privatsphärebedürfnis von Unternehmer*innen" ist für mich relativ einfach.

**Comment by tahesse** [comment]
Es weare doch auch moeglich einen API Token bei Registrierung zu issuen? Dann waeren die Daten nicht direkt zu holen, sondern es muesste zuerst eine Registrierung abgeschlossen werden.

**Comment by Panzer1119** [comment]
> Es geht explizit um die zusätzlichen Dokumente die Informationen enthalten KÖNNTEN die nach dem Gesetz gar nicht erforderlich sind und versehentlich veröffentlicht wurden.

Aber standen nicht all die Daten auch schon lange vorher zur Verfügung (nur halt kostenpflichtig)?
Dann gäbe es da keine zusätzlichen Dokumente, die nicht eh schon öffentlich waren.

Und, dass es so leichter durchsuchbar ist, ist eigentlich egal, da man auch so selber große Datensätze hätte erstellen können, die durchsuchbar sind und das jetzt einfach nur ein Schritt in die Zukunft ist.

**Comment by tahesse** [comment]
@Panzer1119 IMHO handelt der Sachverhalt von der Zugaenglichkeit der Dokumente. Mit einer offenen API koennten Dokumente deutlich einfacher in grossem Volumen maschinell verarbeitet werden koennen.

> Und, dass es so leichter durchsuchbar ist, ist eigentlich egal, da man auch so selber große Datensätze hätte erstellen können, die durchsuchbar sind und das jetzt einfach nur ein Schritt in die Zukunft ist.

Egal, haette und koennte ist kein Gegenargument. Nicht boes gemeint. 😃
Ich denke, dass mit einer Registrierung um einen API Key zu erhalten genug getan ist um den Zugang zu erschweren. So kann man sich auch immer noch vorbehalten _unsachgemaesser Nutzung_ Einhalt zu gebieten.
